### PR TITLE
Migrate pkg/kubelet/cm/cpumanage/{topology/togit pology.go, policy_none.go, cpu_assignment.go} to structured logging

### PR DESCRIPTION
**Pull request by lala123912**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Part of #98976

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
- [KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
```
/sig node instrumentation
/priority important-soon

### CODE REVIEW

File: pkg/kubelet/cm/cpumanager/topology/topology.go [hunk 1]
```diff
@@ -236,8 +236,7 @@ func Discover(machineInfo *cadvisorapi.MachineInfo) (*CPUTopology, error) {
 					}
 				}
 			} else {
-				klog.Errorf("could not get unique coreID for socket: %d core %d threads: %v",
-					core.SocketID, core.Id, core.Threads)
+				klog.InfoS("Could not get unique coreID for socket", "socket", core.SocketID, "core", core.Id, "threads", core.Threads)
```
Comment (ehashman) [inline pkg/kubelet/cm/cpumanager/topology/topology.go:239]:
```suggestion
				klog.ErrorS(nil, "Could not get unique coreID for socket", "socket", core.SocketID, "core", core.Id, "threads", core.Threads)
```

File: pkg/kubelet/cm/cpumanager/cpu_assignment.go [hunk 2]
```diff
@@ -190,7 +190,7 @@ func takeByTopology(topo *topology.CPUTopology, availableCPUs cpuset.CPUSet, num
 	//    on the same sockets as the whole cores we have already taken in this
 	//    allocation.
 	for _, c := range acc.freeCPUs() {
-		klog.V(4).Infof("[cpumanager] takeByTopology: claiming CPU [%d]", c)
+		klog.V(4).InfoS("takeByTopology: claiming CPU", "CPU", c)
```
Comment (ehashman) [inline pkg/kubelet/cm/cpumanager/cpu_assignment.go:193]:
```suggestion
		klog.V(4).InfoS("takeByTopology: claiming CPU", "cpu", c)
```

**Review by ehashman — COMMENTED** [review]
Very close, thanks!

/triage accepted

**Comment by ehashman** [comment]
/lgtm cancel

because we accidentally turned an Errorf into an InfoS, should be ErrorS

File: pkg/kubelet/cm/cpumanager/topology/topology.go (hunk 1 above)
Comment (lala123912) [reply pkg/kubelet/cm/cpumanager/topology/topology.go]: Update to ErrorS.

File: pkg/kubelet/cm/cpumanager/cpu_assignment.go (hunk 2 above)
Comment (lala123912) [reply pkg/kubelet/cm/cpumanager/cpu_assignment.go]: Use `cpu` instead now. Could you review it all again, pls?@ehashman

File: pkg/kubelet/cm/cpumanager/topology/topology.go [hunk 3]
```diff
@@ -236,8 +236,7 @@ func Discover(machineInfo *cadvisorapi.MachineInfo) (*CPUTopology, error) {
 					}
 				}
 			} else {
-				klog.Errorf("could not get unique coreID for socket: %d core %d threads: %v",
-					core.SocketID, core.Id, core.Threads)
+				klog.ErrorS(nil, "Could not get unique coreID for socket", "socket", core.SocketID, "core", core.Id, "threads", core.Threads)
```
Comment (shubheksha) [inline pkg/kubelet/cm/cpumanager/topology/topology.go:239]:
```suggestion
				klog.ErrorS(nil, "Could not get unique coreID for socket", "socketID", core.SocketID, "coreID", core.Id, "threads", core.Threads)
```

Comment (ehashman) [reply pkg/kubelet/cm/cpumanager/topology/topology.go:239]: I don't think this is consistent, we should leave this as is.

**Review by ehashman — COMMENTED** [review]
/lgtm

**Review by mrunalp — APPROVED** [review]

**Pull request merged**